chains/nomad-ethereum/src/gelato/mod.rs

```diff
@@ -0,0 +1,218 @@
+use ethers::signers::Signer;
+use ethers::types::transaction::eip2718::TypedTransaction;
+use ethers::types::{Address, H256};
+use ethers::{prelude::Bytes, providers::Middleware};
+use gelato_relay::{GelatoClient, RelayResponse, TaskState};
+use nomad_core::{ChainCommunicationError, Signers, TxOutcome};
+use std::{str::FromStr, sync::Arc};
+use tokio::task::JoinHandle;
+use tokio::time::{sleep, Duration};
+use tracing::info;
+use utils::CHAIN_ID_TO_FORWARDER;
+
+/// EIP-712 forward request structure
```

Comment:
What does this refer to?

Reply:
Yeah I need to document this better. A ForwardRequest is what we send to Gelato. It encodes the tx we want them to execute as well as the proper signature and chain data they need for their proxy contract to accept it. Will add docs

Reply:
/// comment is for the types module though